compiler/src/iree/compiler/DispatchCreation/test/dispatch_region_formation_preprocessing.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(iree-dispatch-creation-preprocessing-pipeline)" %s | FileCheck %s
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(iree-dispatch-creation-preprocessing-pipeline{const-expr-hoisting})" %s | FileCheck %s
```

Comment:
I think the current state is that the option should be `true` by default, so we shouldnt need this change.